internal/config/instance.go

```diff
@@ -231,6 +231,10 @@ func (i *Instance) ReadInConfig() error {
 
 	err = pl.WaitForLock(5 * time.Second)
 	if err != nil {
+		lockedErr := &lockfile.AlreadyLockedError{}
+		if errs.Matches(err, &lockedErr) {
+			return errs.Wrap(err, "Could not write config as another process appears to be using it")
```

Comment:
Why did you just detect the error type just to wrap it with a regular error? Did you intend to localize this?

Reply:
No, importing `locale` in this package causes an import cycle. The intent here was to add some more context to the error. Do these wrapped messages not bubble up to the user in the same way as the localized errors? If so, I can remove this if it won't be helpful.

Reply:
At a glance, I think it makes sense to not return localized errors from this package, and then have handlers provide localized errors.

If that's not acceptable, I suggest that locale is doing too much by affecting persistent application state.

Reply:
Non localized errors do not bubble up to the user. And the error chain is its own context. I think we can drop this.